app/src/main/res/layout/country_preview_card.xml

```diff
@@ -0,0 +1,120 @@
+                android:id="@+id/countryFlag"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:scaleType="centerCrop" />
+
+        </androidx.cardview.widget.CardView>
+
+        <TextView
+            android:id="@+id/textView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:text="@string/country_label"
+            android:textColor="@color/labels_text_color"
+            android:textSize="@dimen/labels_small_text_size"
+            app:layout_constraintStart_toEndOf="@+id/cardView"
+            app:layout_constraintTop_toTopOf="@+id/cardView" />
+
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/country_name"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="4dp"
+            android:ellipsize="end"
+            android:maxLines="1"
+            android:textColor="@color/primary_text_color"
+            android:textSize="@dimen/country_name_text_size"
+            android:textStyle="bold"
+            app:autoSizeMinTextSize="@dimen/autosize_min_text_size"
+            app:autoSizeTextType="uniform"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="@+id/textView"
+            app:layout_constraintTop_toBottomOf="@+id/textView" />
+
+        <TextView
+            android:id="@+id/textView2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:text="@string/capital_label"
+            android:textColor="@color/labels_text_color"
+            android:textSize="@dimen/labels_small_text_size"
+            app:layout_constraintBottom_toBottomOf="@+id/cardView"
+            app:layout_constraintStart_toEndOf="@+id/cardView"
+            app:layout_constraintTop_toTopOf="@+id/cardView"
+            app:layout_constraintVertical_bias="0.42" />
+
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/country_capital"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="4dp"
+            android:ellipsize="end"
+            android:maxLines="1"
+            android:textColor="@color/primary_text_color"
+            android:textSize="@dimen/country_info_normal_text_size"
+            android:textStyle="bold"
+            app:autoSizeMinTextSize="@dimen/autosize_min_text_size"
+            app:autoSizeTextType="uniform"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="@+id/textView2"
+            app:layout_constraintTop_toBottomOf="@+id/textView2" />
+
+        <TextView
+            android:id="@+id/textView3"
```

Comment:
Let's give it some meaningful name also.

Reply:
Done.